Source/WebCore/platform/mediastream/gstreamer/RealtimeOutgoingMediaSourceGStreamer.cpp

```diff
@@ -42,11 +42,13 @@ RealtimeOutgoingMediaSourceGStreamer::RealtimeOutgoingMediaSourceGStreamer(const
 {
     m_bin = gst_bin_new(nullptr);
 
+    m_inputSelector = gst_element_factory_make("input-selector", nullptr);
```

Comment:
`makeGStreamerElement`?

Reply:
No, this is a core element, I thought we agreed to use makeGStreamerElement for non-core elements only :)